table/block_based/block.h

```diff
@@ -150,8 +150,7 @@ class BlockReadAmpBitmap {
 class Block {
  public:
   // Initialize the block with the specified contents.
-  explicit Block(BlockContents&& contents, SequenceNumber _global_seqno,
-                 size_t read_amp_bytes_per_bit = 0,
+  explicit Block(BlockContents&& contents, size_t read_amp_bytes_per_bit = 0,
```

Comment:
> The problem here (not with the PR per se but with the pre-existing default parameter values) is that removing `_global_seqno` changes the meaning of code like this:
> 
> ```
> Block reader1(std::move(contents), kDisableGlobalSequenceNumber); // kDisableGlobalSequenceNumber is now read_amp_bytes_per_bit !!!
> ```
> 
> We should review all places where `Block`s are constructed and fix any problems like this.

Good catch. I went through all occurrences of `Block` and found a bunch. You can see the fixes in the "address comments" commit (https://github.com/facebook/rocksdb/pull/6429/commits/35be04db05b21e37638cf4d70e3e3fd3759df0b8).